Tools/autotest/common.py

```diff
@@ -1552,6 +1572,7 @@ def run_one_test(self, name, desc, test_function, interact=False):
             tee = None
             self.check_sitl_reset()
             return
+        self.test_timings[desc] = time.time() - start_time
         self.context_pop()
         self.progress('PASSED: "%s"' % prettyname)
```

Comment:
maybe print the time here too , so we don't have to scroll down to check elasped time with reviewing travis log